api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -70,6 +79,17 @@ public UserService(Provider<User> userProvider,
     this.complianceService = complianceService;
   }
 
+  /**
+   * Provides unfettered access to the DAO instance that sits beneath this service class.
+   *
+   * This accessor is meant primarily for tests, or scenarios where a controller needs to bypass
```

Comment:
Is this entirely necessary? Is there a reason you can't just `@Autowire UserDao` straight into your unit test?

Reply:
Yes, I noticed this during a recent review. The existence of that method felt like it might be breaking some abstractions, but I didn't do any deeper investigation into the merits of that approach.